_source/logzio_collections/_metrics-sources/kubernetes.md

```diff
@@ -75,6 +75,11 @@ kubectl --namespace=kube-system create secret generic cluster-details \
 ```shell
 kubectl --namespace=kube-system create -f https://raw.githubusercontent.com/logzio/logz-docs/master/shipping-config-samples/k8s-metricbeat.yml
 ```
+If some of your metrics are not arriving and you see this error in your welcome dashboard - (Ask Daniel Tekatch), run the command below instead. The cosequences of using this yaml is - (Ask Josh Sch)
```

Comment:
@daniel-tk What you got for me on this one?

Reply:
I'd phrase it like this:
"if it looks like not all of your metrics are arriving, if some Kubernetes dashboards are not showing any data, or if some of your documents contain the field 'error.message' and has the following error message: "error making http request: Get https://<<IP Address>>/stats/summary: x509: cannot validate certificate for <<IP Address>> because it doesn’t contain any IP SANs", than run command below instead...